lib/database_config.py

```diff
@@ -9,15 +9,33 @@
 
 
 def get_database_config(development_mode=False):
+    # the following options are validated to get database credentials
```

Comment:
This is a docstring not a comment, so use triple-quote syntax.

I would also say:

To get database credentials, we check and validate these options in order:
...

We _could_ also mention that VCAP and DATABASE_URL are env vars, because this might not be clear to a newcomer

Reply:
Change to triple-quote. If people are checking build pack code I will assume they know what a VCAP is.